docs/antrea-gke.md

```diff
@@ -0,0 +1,128 @@
+# Deploying Antrea on a GKE cluster
```

Comment:
maybe rename this file to gke-installation.md to be consistent with eks-installation.md which is already merged

Reply:
Done